app/core/tests/test_models.py

```diff
@@ -118,3 +118,8 @@ def test_affiliation_is_neither_partner_and_sponsor(affiliation4):
 def test_check_type(check_type):
     assert str(check_type) == "This is a test check_type."
     assert check_type.description == "This is a test check_type description."
+
+
+def test_soc_major(soc_major):
+    assert str(soc_major.title) == "Test Soc Major"
+    assert str(soc_major.occ_code) == "22-2222"
```

Comment:
```suggestion
    assert str(soc_major) == "Test Soc Major"
```
It looks like you're actually testing the python `str` function rather than the model's `__str__` function, since you're passing a stiring (`soc_major.title`) into the `str()` function. You should write it this way so it calls the `__str__` function of the object.